hugo/content/downloads.md

```diff
@@ -139,14 +139,15 @@ The most recent snapshot version is the first snapshot of Mumble version 1.4.0
 Note that we are currently not able to provide static server binaries for Linux or macOS nor are we able to provide a snapshot PPA for Linux just yet.
 
 {{< content-layout/downloads >}}
-{{< content-layout/download name="Windows Client 64-bit" href="https://dl.mumble.info/snapshot/Mumble-1.4.0~2020-12-24~gdb4b6be~snapshot.winx64.msi" osclass="windows">}}
-{{< content-layout/download name="Windows Server 64-bit" href="https://dl.mumble.info/snapshot/Murmur-1.4.0~2020-12-24~gdb4b6be~snapshot.winx64.msi" osclass="windows">}}
-{{< content-layout/download name="Windows Client 32-bit" href="https://dl.mumble.info/snapshot/Mumble-1.4.0~2020-12-24~gdb4b6be~snapshot.msi" osclass="windows">}}
-{{< content-layout/download name="Windows Server 32-bit" href="https://dl.mumble.info/snapshot/Murmur-1.4.0~2020-12-24~gdb4b6be~snapshot.msi" osclass="windows">}}
+{{< content-layout/download name="Windows client (x64)" href="https://dl.mumble.info/latest/snapshot/client-windows-x64" osclass="windows">}}
+{{< content-layout/download name="Windows server (x64)" href="https://dl.mumble.info/latest/snapshot/server-windows-x64" osclass="windows">}}
+{{< content-layout/download name="Windows client (x86)" href="https://dl.mumble.info/latest/snapshot/client-windows-x86" osclass="windows">}}
+{{< content-layout/download name="Windows server (x86)" href="https://dl.mumble.info/latest/snapshot/server-windows-x86" osclass="windows">}}
 {{< /content-layout/downloads >}}
 
 {{< content-layout/downloads >}}
-{{< content-layout/download name="macOS" href="https://dl.mumble.info/snapshot/Mumble-1.4.0~2020-12-24~gdb4b6be~snapshot.dmg" osclass="mac">}}
+{{< content-layout/download name="macOS client >= 10.13 (x64)" href="https://dl.mumble.info/latest/snapshot/client-macos-x64" osclass="mac">}}
+{{< content-layout/download name="macOS client <= 10.12 (x64)" href="https://dl.mumble.info/latest/snapshot/client-macos-hfs+-x64" osclass="mac">}}
```

Comment:
I intentially wrote the macOS version directly after "macOS" as this way I think could be ambiguous: people could interpret this as the client version instead of the OS version

Reply:
Ah, sorry.

What about `Client for macOS >= 10.13 (x64)`?

Reply:
Yep that'd be fine. A bit inconsistent with the other platforms but I guess it doesn't matter

Reply:
I will of course change the string for the other platforms as well.

Reply:
Done!